docs/Settings.md

```diff
@@ -65,6 +65,7 @@ The following settings are available via registry as well as via [QUIC_SETTINGS]
 | Congestion Control Algorithm       | uint16_t   | CongestionControlAlgorithm  |         0 (Cubic) | The congestion control algorithm used for the connection.                                                                     |
 | ECN                                | uint8_t    | EcnEnabled                  |         0 (FALSE) | Enable sender-side ECN support.                                                                                               |
 | Stream Multi Receive               | uint8_t    | StreamMultiReceiveEnabled   |         0 (FALSE) | Enable multi receive support                                                                                                  |
+| Opt in to use QTIP              | uint8_t    | QTIPEnabled   |         0 (FALSE) | (CAUTION: for advanced users only) When set to true at a global level, all listeners and client connections (must be running in XDP mode) created thereafter will initialize a TCP socket and attempt to bind to the local address of the listener/client connection. This is to reserve a TCP port for your listener/client connection to ensure XDP does not steal any TCP traffic from other processes later. That also means you need to ensure no other processes are listening on the same TCP port as the local address of your listener prior to starting your listener. When client connections are started with QTIPEnabled set to true, they will exclusively send/recv data over QTIP. For listeners, they are able to recv/send data over both QTIP+XDP or QUIC+XDP, depending on the connection. |
```

Comment:
We should have a doc purely on QTIP, and point to that for more details. This should be short and sweet.

Reply:
Put details in it's own QTIP.md file.